<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Set Variable Names / Equations</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row0">
       <item>
        <widget class="QLineEdit" name="var0name">
         <property name="text">
          <string>RHO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="var0form"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLineEdit" name="var1name">
         <property name="text">
          <string>PPP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="var1form"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row2">
       <item>
        <widget class="QLineEdit" name="var2name">
         <property name="text">
          <string>UU1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="var2form"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="var3name">
         <property name="text">
          <string>UU2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="var3form"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="var4name">
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="var4form"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="var5name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="var5form"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="var6name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="var6form"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row7">
       <item>
        <widget class="QLineEdit" name="var7name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="var7form"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
